testing/microlith-test/integration/prometheus_basic_auth/prometheus.bats

```diff
@@ -14,52 +14,64 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-load "$BATS_DETIK_ROOT/utils.bash"
-load "$BATS_DETIK_ROOT/linter.bash"
-load "$BATS_DETIK_ROOT/detik.bash"
 load "$BATS_SUPPORT_ROOT/load.bash"
 load "$BATS_ASSERT_ROOT/load.bash"
 load "$BATS_FILE_ROOT/load.bash"
 
-export SERVER_IMAGE=${SERVER_IMAGE:-couchbase/server:7.0.1}
-export CMOS_IMAGE=${CMOS_IMAGE:-couchbase/observability-stack:v1}
-export CMOS_PORT=${CMOS_PORT:-8080}
-
 setup() {
-    if [ "$TEST_INTEGRATION" == "true" ]; then
-        skip "Skipping integration tests"
+    if [ "$TEST_NATIVE" != "true" ]; then
+        skip "Skipping native prometheus tests"
     fi
 }
 
 teardown() {
-    if [ "$SKIP_TEARDOWN" == "true" ]; then
```

Comment:
Would be good to keep this around - makes debugging failures locally much easier

Reply:
Sure, need to make sure we're consistent as well across the board. I'm not sure if there is an existing skip teardown approach in BATS though which would be better...